.github/workflows/ci-lint.yaml

```diff
@@ -48,6 +51,14 @@ jobs:
           towncrier check --compare-with origin/${BASE_BRANCH}
         env:
           BASE_BRANCH: ${{ github.base_ref }}
+        if: |
+          github.event_name == 'pull_request' &&
+          github.actor != 'dependabot[bot]'
+      - name: Lint git commits
+        run: |
+          make lint-git
```

Comment:
I hate lint-git :cry: , it's pedantic for very little payoff. Mostly beneficial for people reading patches in emails in Mutt, Linus style. </rant>

Do we have an autoformat tool, ideally in conjunction with commit hooks? This is not a request for one, just trying to see if there's existing tooling support.

Reply:
I hate incoming `lint-docs` even more. Anyway, I will double check if this is a hard requirement.

None of a repo I am familiar with is triggering commit hooks to check gitlint. Tbh, I hit that error once this year, so commit hook would be more painful.

Reply:
Org level decision is: gitlint and markdownlint should be the same. Golint is subject for per repo config

Reply:
Yeah I resigned to having the org-wide configs a long time ago.
FWIW, my personal philosophy is that autoformat is good (brings consistency at the minor cost of odd formatting choices sometimes), whereas enforced syntax-only lint without autoformat is bad (brings consistency but requires masochism). But this is not the place for that discussion :)

Anyway, in my experience they can be automated away pretty well with autoformatting. I might invest some time into that, proportional to how much the new lints annoy me. Thanks for the info on what already exists (which is not much, sounds like.)

Reply:
This looks potentially helpful for markdownlint, IDK if you're using it already: https://github.com/DavidAnson/vscode-markdownlint/blob/main/README.md#fix

Reply:
yes it is helpful, but markdownlint cannot fix MD013 which is the most common issue https://github.com/DavidAnson/markdownlint/issues/535

Reply:
I'm preparing autoformatting with `prettier` in #666. I don't know if it does a perfect job of breaking lines (though I do think it has markdown awareness), but almost certainly it's good enough for the great majority of these simple short texts: commit messages and changelog fragments.